package/lite/etc/conf.d/conflib/raw/app-raw-bsd_nopri.conf

```diff
@@ -53,7 +53,7 @@ block parser app-raw-bsd_nopri() {
 };
 application app-raw-bsd_nopri[sc4s-raw-syslog] {
     filter {
-        message('^\w\w\w \d\d \d\d:\d\d:\d\d ');
+        message('^\w\w\w [ \d]\d \d\d:\d\d:\d\d ');
```

Comment:
Same comment

Reply:
Updated the regex with some more constraints.